src/index.tsx

```diff
@@ -450,6 +446,10 @@ interface Options extends BaseOptions {
   patchedOrientationAndroid?: string;
 }
 
+export function lookUpLocation(location: Location): LocationLookup {
```

Comment:
TODO for Johannes make an issue to figure out how to run this function in a react-native-worklets-core Worklet and not having to expose it outside the plugin.

Reply:
https://linear.app/inaturalist/issue/ML-187/refactor-plugin-to-run-location-lookup-only-internally